server/backend/api/app/routers/router_application.py

```diff
@@ -43,3 +49,36 @@ def get_fam_application_user_role_assignment(
                  # of results = {len(app_user_role_assignment)}"
     )
     return app_user_role_assignment
+
+
+@router.get(
+    "/{application_id}/users/{user_id}",
+    response_model=FamUserInfoSchema,
+    status_code=200,
+    dependencies=[Depends(authorize_by_app_id)],
+    summary="Retrieve User Information by User ID under an application",
+    responses={
+        404: {
+            "description": "User not found",
+            "content": {"application/json": {"example": {"detail": "User not found"}}},
+        }
+    },
+)
+async def get_user_by_user_id(
```

Comment:
Since this is under "_application" context for the resource, might be better to name it like "get_application_user_by_id".

Reply:
That does sound better, changed here
https://github.com/bcgov/nr-forests-access-management/pull/1607/commits/ead5ad2d0dcfdf292997c3bc2f6ab932bad76a29